data/korath/korath ships.txt

```diff
@@ -851,6 +851,213 @@ ship "Korath World-Ship" "Korath World-Ship C"
 	turret 91 290 "Korath Warder"
 
 
+ship "Korath Haven"
+	sprite "ship/korath haven"
+	"uncapturable"
+	attributes
+		category "Heavy Freighter"
+		cost 27690000
+		shields 196420
+		hull 320690
+		"required crew" 1940
+		"bunks" 10920
+		"mass" 12000
+		"drag" 84
+		"heat dissipation" .22
+		"fuel capacity" 1700
```

Comment:
Even though it's a superheavy that uses fuel weapons, I don't think it needs this much.  That's what Fuel Processors are for.

```suggestion
		"fuel capacity" 1000
```